### ISSUE TITLE
How to run without compiling?

### ISSUE THREAD

**Issue by dogweather** [joakin/elm-node#1]
Excellent library! Thank you.

I haven't figured out, though, how to just run a compiled `.js` file. Running it with `node`, for example, doesn't show the `log` output. And successively running `elm-node` recompiles the source file even if it hasn't changed.

Is there a way to run w/out compiling?

**Comment by joakin** [comment]
Can you share how you are trying to run the JS?

If I remember correctly there should be the output file to run in a folder
so it should be possible to run it directly.

**Comment by joakin** [comment]
I think there should be a `elm-node-stuff/index.js` that will run the node program.

**Comment by joakin** [comment]
I've published v1.1.5 with a console log showing the node.js entrypoint, so now it looks like this:

```
elm-node-test → elm-node --example-elm > src/Main.elm
elm-node-test → elm-node src/Main.elm --optimize
Success!

    Main ───> elm-node-stuff/elm.js

    node.js entrypoint ───> elm-node-stuff/index.js

Main application initialized
elm-node-test → node elm-node-stuff/index.js
Main application initialized
```

Hopefully that helps
